hangman_dinner.rb

```diff
@@ -0,0 +1,201 @@
+require "colorize"
+
+class Hangman
+    def initialize
+
+      @answer_bank = ["cupcake", "spaghetti", "cookie", "hamburger", "pizza", "sandwich", "crackers", "salad","cheese", "burrito"]
+      @guesses = 4
+      @num_correct = 0
+      @current_answer = @answer_bank[rand(@answer_bank.length)]
+      @current_answer_array = @current_answer.split(//)
+      @fillable_answer = []
+      @alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K","L", "M", "N", "O", "P", "Q", "R", "S",
+      "T", "U", "V", "W", "X", "Y", "Z"]
+      @already_guessed =[]
+      @current_answer_array.length.times do |x|
+        @fillable_answer[x]= " _ "
+      end
+
+
+      draw_art
+      print_word
+      play
+
+    end
+
+    def draw_art
+
+
+      art = <<-END
```

Comment:
For future reference, when you're defining multiline strings like this, the text token doesn't have to be `END`, you can so anything you want. So in this case, `ART` would make it more clear.